workspaces/ui/src/contexts/InitialRfcCommandsContext.js

```diff
@@ -2,25 +2,30 @@ import * as React from 'react';
 import {GenericContextFactory} from './GenericContextFactory.js';
 
 const {
-    Context: InitialRfcCommandsContext,
-    withContext: withInitialRfcCommandsContext
-} = GenericContextFactory(null)
-
+  Context: InitialRfcCommandsContext,
+  withContext: withInitialRfcCommandsContext
+} = GenericContextFactory(null);
+global.devlog2 = []
 class InitialRfcCommandsStore extends React.Component {
-
-    render() {
-        const {rfcId, initialCommandsString, initialEventsString} = this.props
-
-        return (
-            <InitialRfcCommandsContext.Provider value={{rfcId, initialCommandsString, initialEventsString}}>
-                {this.props.children}
-            </InitialRfcCommandsContext.Provider>
-        )
-    }
+  render() {
+    const {rfcId, initialCommandsString, initialEventsString} = this.props;
+    global.devlog2.push({
```

Comment:
we can remove this. was helpful for me for debugging though, via watching console.table(devlog2)